### ISSUE TITLE
Please make parenthesis more readable

### ISSUE THREAD

**Issue by sidhuharry** [polarsys/b612#25]
The parenthesis () are very similar to the square brackets. This is causing readability issues because () looks like []. Function calls look like array declaration.

**Comment by georgalis** [comment]
Concur. What an excellent font, however B612Mono programming use is prohibited due to similarity of these characters:<pre>;: '` ,. ({[ )}] O0</pre>  The consequence of transposing any of these in software would be catastrophic.

If you are able to infer the characters from context, their homogeneity might promote legibility and comfort of reading; however, the criticality to distinguish transposition of these characters, makes B612Mono impossible to use for technical composition.

**Comment by anilkumar150886** [comment]
Yes.. I agree.. Such a wonderful font.. but it's very difficult to distinguish b/w the below characters
:  -> ;
' -> " ->`
, -> .
o->O->0
small brackets (). Currently it looks like square brackets [ ]

**Comment by joby-lol** [comment]
I agree. I really like most of this font, but I guess I can't use it as my text editor font because it's impossible to tell the difference between parentheses and square brackets.

**Comment by georgalis** [comment]
for reasons I can only speculate, intel-one-mono is not available from fonts.google.com
however once you acclimate to the highly distinctive characters,
it is a breeze to read even in smaller sizes. 

Proportional, B612 https://b612-font.com/
designed and tested to be used on aircraft cockpit screens

For programming, the "B612 Mono" font has problems, but this one is excellent...

Monospace, Intel One Mono https://github.com/intel/intel-one-mono
for maximum legibility to address developers' fatigue
and eyestrain and reduce coding errors.

**Comment by anilkumar150886** [comment]
@georgalis thanks for the wonderful font, is there a way i can rename the font name to something like 1-InputMono ? Coz, i want to be in the start of font list.. Please let me know.

Thanks

**Comment by anilkumar150886** [comment]
I'm trying to do it in https://www.glyphrstudio.com/online/. But somehow, its changing the width of the glyphs to bigger size..

**Comment by georgalis** [comment]
@anilkumar150886 just use latex then name it whatever you want

**Comment by anilkumar150886** [comment]
hmm.. i'm sorry, i did not understand.. i wanted to change the ttf font name itself to 1-InputMono.. Will you be able to help me out ..?

**Comment by Firstyear** [comment]
B612Mono-Regular-Tweaked.ttf from https://github.com/Firstyear/b612/tree/master/fonts/ttf now has fixed braces, period and others. It also has 0 with / by default as the numerical 0. Hope this helps!
